<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DelaysDialog</class>
 <widget class="QDialog" name="DelaysDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>171</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>321</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>321</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SPI Delays</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/active64.png</normaloff>
    <normalon>:/icons/icons/active64.png</normalon>
    <disabledoff>:/icons/icons/greyed64.png</disabledoff>
    <disabledon>:/icons/icons/greyed64.png</disabledon>
    <activeoff>:/icons/icons/active64.png</activeoff>
    <activeon>:/icons/icons/active64.png</activeon>
    <selectedoff>:/icons/icons/selected64.png</selectedoff>
    <selectedon>:/icons/icons/selected64.png</selectedon>:/icons/icons/active64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCSToDataDelay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>17</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Chip select to data (100 μs):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxCSToDataDelay">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelDataToCSDelay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>57</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Data to chip select (100 μs):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxDataToCSDelay">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelInterByteDelay">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>97</y>
     <width>195</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Inter-byte (100 μs):</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxInterByteDelay">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DelaysDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DelaysDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>160</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
